app-emulation/docker/files/dockerd

```diff
@@ -76,7 +77,7 @@ if [[ -z "${ARG_DRIVER}" && -z "${DOCKER_DRIVER}" ]]; then
 fi
 
 # Enable selinux except when known to be unsupported (btrfs).
-USE_SELINUX="--selinux-enabled"
-maybe_disable_selinux
+maybe_enable_selinux
+
+exec docker "$@"
 
-exec docker "$@" "${USE_SELINUX}"
```

Comment:
It should be sufficient to just remove the quotes from around `${USE_SELINUX}`. I'm curious if the quotes around `$@` are also going to be potentially problematic.